### DISCUSSION TITLE
Update icon library

### DISCUSSION THREAD

**Discussion by diegoleme** [directus/directus#14374] [Archive]
I'm not able to use some icons that exist in the latest version of the Material. Eg: barcode.

Couldn't we use the font directly from Google Fonts to always have the latest version?

**Comment by azrikahar** [comment]
I believe the one with [`barcode`](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Abarcode%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048) is actually Material Symbols, not the current Material Icons included in Directus, which technically isn't just "the latest version of the Material" (as in a new rework vs just bumping a version number).

This has been primarily discussed in #12958, so feel free to keep this discussion over there 👍 I believe one main thing to be mindful of is to ensure not to break any existing usage of `_outline` icons by users or the app itself.

**Reply by rijkvanzanten** [reply]
Too bad we can't really rely on the google-provided CDN link for the font (that would keep it auto-updated) due to GDPR issues 🙃

**Reply by azrikahar** [reply]
Maybe we can use something like https://github.com/fontsource/fontsource 🤔 Though Material Symbols are not added yet but tracked in https://github.com/fontsource/fontsource/issues/497.

(or maybe [`@material-design-icons/font`](https://www.npmjs.com/package/@material-design-icons/font)/[`material-symbols`](https://www.npmjs.com/package/material-symbols) which seems to be updated on a schedule via github actions)

**Reply by rijkvanzanten** [reply]
At that point, you're still self-hosting the font, so we'd have the same issue 🙂 

I think for now we should upgrade to material-symbols (instead of icons), and then setup some sort of automation to keep it up-to-date, or set a calendar event every month as a reminder
